Makefile

```diff
@@ -20,33 +20,33 @@ decrypt-previous-staging:
 	@cd env/staging &&\
 	curl -L -o .previous.env.zip.enc.aws https://github.com/cds-snc/notification-manifests/blob/main/env/staging/.env.zip.enc.aws?raw=true > /dev/null 2>&1 &&\
 	aws kms decrypt --ciphertext-blob fileb://.previous.env.zip.enc.aws --output text --query Plaintext --region ca-central-1 | base64 --decode > .previous.env.zip &&\
-	unzip -o .previous.env.zip
+	unzip -o .previous.env.zip && mv .env .previous.env
```

Comment:
unzip writes to the original file name, so we have to move it after the unzip and be careful later to do `decrypt-previous-staging` before `decrypt-staging`